common/autoware_ad_api_msgs/srv/InterfaceVersion.srv

```diff
@@ -0,0 +1,6 @@
+---
+autoware_ad_api_msgs/ResponseStatus status
+string version
+uint32 major
+uint32 minor
+uint32 patch
```

Comment:
Do we need `uint32` for these values?
I feel `uint8` or `uint16` is enough for version numbers.

Reply:
OK, use `uint16` instead. https://github.com/autowarefoundation/autoware.universe/pull/704/commits/a8985884916eeffe94d1c4d338593b5e4879415e